Separate readme and logs

CFGrote opened the pull request:
This PR restructures the content of the top level readme and performs various cleanups

- Convert readmes to org and transclude from run logs.
- Remove readme in Run 1 logdir.
- Replace .md by .org, update all plots, move all log to one logbook.
- Remove more mds and document generation of reduced size graphs.
- Update logbook
- Update readme, logbook
- Update log

Commit:
Convert readmes to org and transclude from run logs.

Commit:
Remove readme in Run 1 logdir.

Commit:
Replace .md by .org, update all plots, move all log to one logbook.

Commit:
Remove more mds and document generation of reduced size graphs.

Commit:
Update logbook

- add Run 1

Commit:
Update readme, logbook

- add link to logbook into readme.

Commit:
Update log

- remove duplicate figure
- cleanup heading levels

marimeireles commented on queries/20241210T091800_analyze_timings.ipynb:
I've never done this before so I'm really not sure how to review `ipynb`s. Not sure if there's a good way?
In `20241210T091800_analyze_timings`, at this cell:
```
ax.set_xlabel('')
ax.set_ylabel('time')
ax.set_title("Query "+DATASET.replace("_", " ").replace("-", ": "))ax
```
`ax` is being used before it's defined, I think it's a matter of rearranging cells.
There's also an empty cell in this notebook.

CFGrote replied:
i cleaned up the notebooks and removed unneeded cells.

marimeireles commented on Logbook.org:
```diff
@@ -0,0 +1,293 @@
+- client software: apache-jena-5.2.0/bin/rsparql
+- rsparql version reported as 4.2.0
+- connection: vpn to uni-muenster
+- note: all queries were run once about 30min before. Noting reduced
+  query time on the 00-construct and other queries. Caching?
```
We can add there was no available API for controlling caching within servers so the best we could was hot cache?

CFGrote replied:
done in 32df07f

marimeireles commented on Logbook.org:
```diff
@@ -0,0 +1,293 @@
+:PROPERTIES:
+:ID:       825c8799-bb59-4b89-aa22-1b4b2d1279c8
+:END:
+By changing the order with respect to Run1, we try to eliminate the effect of caching, as each server never sees the same two queries immediately one after the other, wherea
```
grammar:
wherea -> whereas
and erase line break

CFGrote replied:
done in 32df07f

marimeireles commented on Logbook.org:
```diff
@@ -0,0 +1,293 @@
+     :CUSTOM_ID: results-1
+     :ID:       e49db261-4cf9-4b44-823c-6045b3984f17
+     :END:
+Timings come out differently: for many queries, now fuseki is faster,
+ontop factor 2 slower.
+[[file:queries/saved_runs/20241210T113800/facet_walltime.png]]
```
Any idea why?

CFGrote replied:
Well this was what we expected, right? ontop has the additional task to rewrite a received sparql query into sql and re-translate the sql response into rdf.

CFGrote replied:
added short discussion to Run 2 section in commit 906cb3e

marimeireles reviewed (commented):
Just minor comments.

Commit:
Cleanup notebooks as per @marimeireles' comments on PR #4.

Commit:
Update log reflecting @marimeireles' comments in PR #4.

Commit:
Update Logbook.

- Add discussion on factor 2 in query response time ontop vs. fuseki (Run 2) in
response to @marimeireles's comment in PR #4.

Commit:
Adding results and log for run 5 (issue #13)
